client/pages/profile.jsx

```diff
@@ -44,6 +45,53 @@ const Profile = (props) => {
       .catch((err) => console.error(err));
   };
 
+  const renderFollow = (data) => {
+    return (
+      <div className="row flex-column profile-brown justify-center profile-egg-icons-div">
+        <ul className="events-ul">
+          {data.map((data) => (
+            <li
+              key={data.username + Math.random()}
```

Comment:
Instead of using a random number, you can just use the index of the array element. Also, `data` is not really a good name. these are `events` and you are mapping over each `event`.

```js
events.map((event, index) => {

// use index as the key
```